cms/static/js/spec/views/pages/course_outline_spec.js

```diff
@@ -944,18 +958,21 @@ define(["jquery", "common/js/spec_helpers/ajax_helpers", "common/js/components/u
                                     "is_time_limited": false,
                                     "is_practice_exam": false,
                                     "is_proctored_exam": false,
-                                    "default_time_limit_minutes": 150
+                                    "default_time_limit_minutes": 150,
+                                    "hide_after_due": false,
```

Comment:
Yes, it will. It's a field on the `ProctoredExam` model, and is stored with the exam definition.